compose/completion.md

```diff
@@ -9,6 +9,16 @@ for the bash and zsh shell.
 
 ## Install command completion
 
+### Fig
+
+<a href="https://fig.io/" target="_blank"><img src="https://fig.io/badges/Logo.svg" width="15" height="15"/></a> [Fig](https://fig.io/) provides IDE-style autocompletions for Docker. It works in bash, zsh, and fish.
```

Comment:
Could you remove the hot-linked logo, and add a `target` and `rel=noopened` on the line to the Fig website?

```suggestion
[Fig](https://fig.io/){:target="_blank" rel="noopener"} provides IDE-style autocompletions for Docker. It works in bash, zsh, and fish.
```

Reply:
@thaJeztah Thanks so much for having a look at this. 
We might recover this page together with this feature in the upcoming future. For now, in line with the decision about the completion page, I'm closing this PR instead of leaving it open as there's no clear timeline. 
@brendanfalk thanks for your contribution, feel free to reopen this PR if this feature becomes available again in the near future.

Reply:
Hey! I'm so incredibly sorry for the massive delays here. I completely missed the above comments. 

Very happy to merge with those changes by @thaJeztah above. Would this enable Fig to be merged?

> For now, in line with the decision about the completion page, I'm closing this PR instead of leaving it open as there's no clear timeline.

Not sure I fully follow this. Was there a decision somewhere else that I missed? Or is it the lack-of a final decision that is causing it to be closed?

Reply:
Ahh. My apologies. I now see the completions.md file has been removed. Makes total sense.

Thanks for the commentary. We will keep a close eye on things and be sure to contribute if/when things change again!